### ISSUE TITLE
WiredTiger - Can't list collections

### ISSUE THREAD

**Issue by remialvado** [bobthecow/genghis#209]
I've tested Genghisapp 2.3.11 and developer branch against MongoDB 3.0.5 using WiredTiger as a backend storage in two configurations : single node and replicaSet with 3 nodes. In both cases, Genghisapp is able to list databases but can't list collections :( 
I've checked Apache logs but can't see anytjing strange
Any idea about what is happening and how I can solve this ?

**Comment by bobthecow** [comment]
Does this happen with and without auth? My suspicion is that it's something permissions-related, but I haven't been able to track it down yet.

**Comment by remialvado** [comment]
It happens without auth

**Comment by remialvado** [comment]
Any idea ? Do you want me to test something to help you debug ?

**Comment by Scorpiion** [comment]
Hi, I think I know why this issue happens, Dart's mongo driver had to be updated to be able to list collections on Mongo 3+. So it's probably the driver, either it's not updated or you need to change the function used.

See the Dart driver issue (by me) here https://github.com/vadimtsushko/mongo_dart/issues/67

TLDR: The driver needs to use [db.getCollectionInfos](http://docs.mongodb.org/manual/reference/method/db.getCollectionInfos/#db.getCollectionInfos) on 3+ to list collections, [the old](http://docs.mongodb.org/manual/reference/system-collections/#%3Cdatabase%3E.system.namespaces) way of doing it is deprecated in 3+.

**Comment by vogre** [comment]
Changing
  spec.add_dependency 'mongo',            '>= 1.8.0', '<= 1.11.1'
to use '1.12.3' (latest 1.x release) fixes it.

**Comment by dioptre** [comment]
+1

**Comment by remialvado** [comment]
I did not reply earlier but found a fix by using mongo driver 1.6.6 instead of driver 1.5.x. As a matter of fact, it seems that there is a new driver for PHP called `mongodb`. In version 1.0, it supports the same thing as `mongo` in 1.6.x but in version 1.1, it will support PHP 7.0 as well as MongoDB 3.2. The legacy `mongo` driver won't be updated anymore.

**Comment by chriskinsman** [comment]
What is the best way to fix 2.3.11?

I am running the gem version on OSX and can't figure out the best way to update the driver for MongoDb 3.2.1.
